### PR TITLE
German translation updates for Git 2.41

### PR DESCRIPTION
**Pull request by ralfth**
Hi team,

please review the German translation updates for Git 2.41.
This time there are 39 updated/new messages.

Thanks

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -12317,22 +12317,23 @@ msgstr ""
...
 
 msgid "when used with --rules-file interpret patterns as cone mode patterns"
 msgstr ""
+"bei Verwendung mit --rules-file werden die Muster als Cone-Modus Muster "
+"interpretiert"
 
-#, fuzzy
 msgid "use patterns in <file> instead of the current ones."
-msgstr "im Index statt im Arbeitsverzeichnis suchen"
+msgstr "Muster aus <Datei> anstelle der aktuellen Muster verwenden"
```
Comment (ruester) [inline po/de.po:12336]: maybe add a `.` at the end like in the original message?

File: po/de.po [hunk 2]
```diff
@@ -15396,6 +15396,7 @@ msgstr "Ungültiger Sektionsname: %s"
 #, c-format
 msgid "refusing to work with overly long line in '%s' on line %<PRIuMAX>"
 msgstr ""
+"Ausführung mit übermäßig langer Zele in ‚%s‘ bei Zeile %<PRIuMAX> verweigert"
```
Comment (ruester) [inline po/de.po]: Zele = Zeile ?
and it should be `'%s'` instead?

File: po/de.po [hunk 3]
```diff
@@ -18457,13 +18457,12 @@ msgstr "Reverse-Index-Datei %s hat nicht unterstützte Version %<PRIu32>"
...
 
-#, fuzzy
 msgid "invalid checksum"
-msgstr "Ungültiger Objekt-Typ %s"
+msgstr "ungültige Prüfsumme"
 
 #, c-format
 msgid "invalid rev-index position at %<PRIu64>: %<PRIu32> != %<PRIu32>"
-msgstr ""
+msgstr "ungültige rev-index Position bei %<PRIu64>: %<PRIu32>!= %<PRIu32>"
```
Comment (ruester) [inline po/de.po]: there is a missing space before `!=`

File: po/de.po [hunk 4]
```diff
@@ -20652,7 +20650,7 @@ msgid ""
 "\n"
 msgstr ""
 "Ausführung erfolgreich: %s\n"
-"Aber Änderungen in Index oder Arbeitsverzeichnis verblieben.\n"
+"Aber es sind Änderungen in Index oder Arbeitsverzeichnis verblieben.\n"
```
Comment (ruester) [inline po/de.po]: "im" instead of "in" maybe?

File: po/de.po (hunk 1 above)
Comment (ralfth) [reply po/de.po:12336]: I think this is wrong in the English message as no other message of that type ends with a dot.

File: po/de.po (hunk 2 above)
Comment (ralfth) [reply po/de.po]: fixed

File: po/de.po (hunk 4 above)
Comment (ralfth) [reply po/de.po]: fixed

File: po/de.po (hunk 3 above)
Comment (ralfth) [reply po/de.po]: fixed

**Comment by ruester** [comment]
Many thanks for your work! 😃 

`Reviewed-by: Matthias Rüster <matthias.ruester@gmail.com>`

**Review by ruester — APPROVED** [review]
Many thanks for your work! 😃 

`Reviewed-by: Matthias Rüster <matthias.ruester@gmail.com>`

**Comment by ralfth** [comment]
Thanks for the review!

**Pull request closed without merging**